2021-01-21 18:34:21,286 INFO    MainThread:215500 [wandb_setup.py:_flush():70] setting env: {}
2021-01-21 18:34:21,286 INFO    MainThread:215500 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-01-21 18:34:21,286 INFO    MainThread:215500 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-21 18:34:21,286 INFO    MainThread:215500 [wandb_init.py:_log_setup():318] Logging user logs to K:\OneDrive - Cumberland Valley School District\Education\Activates\Science Fair\PCG-Science-Fair\PCG-arrhythmia-detection\wandb\run-20210121_183421-2wvdzl7i\logs\debug.log
2021-01-21 18:34:21,286 INFO    MainThread:215500 [wandb_init.py:_log_setup():319] Logging internal logs to K:\OneDrive - Cumberland Valley School District\Education\Activates\Science Fair\PCG-Science-Fair\PCG-arrhythmia-detection\wandb\run-20210121_183421-2wvdzl7i\logs\debug-internal.log
2021-01-21 18:34:24,239 INFO    MainThread:215500 [wandb_run.py:_console_start():1308] atexit reg
2021-01-21 18:34:24,240 INFO    MainThread:215500 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.WRAP
2021-01-21 18:34:24,240 INFO    MainThread:215500 [wandb_run.py:_redirect():1206] Wrapping output streams.
2021-01-21 18:34:24,240 INFO    MainThread:215500 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-21 18:34:24,240 INFO    MainThread:215500 [wandb_watch.py:watch():39] Watching
2021-01-21 18:34:27,156 INFO    MainThread:215500 [wandb_run.py:_config_callback():608] config_cb None None {}
2021-01-21 18:34:31,397 INFO    MainThread:215500 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-21 18:34:31,399 INFO    MainThread:215500 [wandb_run.py:_restore():1250] restore
2021-01-21 18:34:31,694 INFO    MainThread:215500 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "2a69972e1725452791a83147388d66b2"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 878
      total_bytes: 878
    }
  }
}

2021-01-21 18:34:33,709 INFO    MainThread:215500 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "5638dd75f6624ee1be64e0c57cc661bd"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 7753
      total_bytes: 7753
    }
  }
}

2021-01-21 18:34:34,821 INFO    MainThread:215500 [wandb_run.py:_show_files():1614] logging synced files
2021-01-21 18:34:34,826 INFO    MainThread:215500 [internal.py:handle_exit():139] Internal process exited
